SymbolCollisionTest/Podfile

```diff
@@ -12,7 +12,7 @@ target 'SymbolCollisionTest' do
     pod 'FirebaseAnalytics'
     pod 'FirebaseAuth', :path => '../'
     pod 'FirebaseCore', :path => '../'
-#    pod 'FirebaseCrashlytics', :path => '../'  #4746
+    pod 'FirebaseCrashlytics', :path => '../'  #4746
```

Comment:
delete issue number comment. 

Note the SymbolCollisionTest only runs on the travis cron. To test locally in a PR, see https://github.com/firebase/firebase-ios-sdk/tree/master/SymbolCollisionTest#run-locally

Reply:
![Screen Shot 2020-01-28 at 10 44 54 AM](https://user-images.githubusercontent.com/555046/73279493-3ad5c780-41bb-11ea-863b-e9d04873536a.png)

Looks good now